project-service/gulpfile.babel.js

```diff
@@ -0,0 +1,21 @@
+import path from 'path';
+import gulp from 'gulp';
+import shell from 'gulp-shell';
```

Comment:
gulp-shell should be required from within the project-service:test task. Otherwise the project-service:install task is useless...

let shell = require('gulp-shell'); on line 11 will do the trick

can you add your tasks to the root gulpfile?